OptimizelySDK/Odp/LruCache.cs

```diff
@@ -63,6 +62,15 @@ public class LruCache<T> : ICache<T>
         {
             _mutex = new object();
 
+            // TODO: Please add a condition to check minimum value of maxSize as well. 
+            // @msohailhussain What's the minimum value for maxSize?
```

Comment:
@msohailhussain I'd like to better understand the requirement. DM me when you have a moment.

Reply:
@msohailhussain bump

Reply:
@msohailhussain bump

Reply:
Check with @jaeopt